.github/workflows/auto-approve-backports.yml

```diff
@@ -0,0 +1,21 @@
+on:
+  pull_request_target:
+    branches-ignore:
+      - main
+    types:
+      - opened
+
+env:
+  NODE_ENV: kibana-github-action
+
+jobs:
+  approve:
+    name: Auto-approve backport
+    runs-on: ubuntu-latest
+    if: |
+      contains(github.event.pull_request.labels.*.name, 'backport') &&
+      github.event.pull_request.user.login == 'kibanamachine'
+    permissions:
+      pull-requests: write
+    steps:
+      - uses: hmarr/auto-approve-action@v3
```

Comment:
Do we wanna check here if Kibana machine would be able to approve and bypass the required approvals restriction with its own self approval by using

```
with:
          github-token: ${{ secrets. KIBANAMACHINE_TOKEN }}
```

Or have we already discarded that out and wanna move with the github-actions bot?

Reply:
We can try the self-approval, but would it be bad if we use a `github-actions` bot? If it's available, and it does the job, it's probably good. But I'm okay, we can do a first try with the self-approval.